Project-Descriptions-and-Plans/CV32E4_TBD/Project_Concept_for_CV32E4_TBD_Nov-27_2023.md

```diff
@@ -0,0 +1,250 @@
+# OpenHW Project Concept Proposal: CV32E4_TBD
+*The PC proposal introduces the project and explains the market drivers and the "why"*
+
+# Title of Project - "CV32E4_TBD"
+# Project Concept Proposal
+## Date of proposal - 2023-11-27
+## Author(s) - Manuel PEZZIN (CEA)
+
+## High Level Summary of project, project components, and deliverables
+This proposal is a continuation of CV32E40Pv2 project.
```

Comment:
this creates confusion, cause you are actually making a new project as it is not a v3 project -